.github/workflows/ci.yml

```diff
@@ -0,0 +1,129 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
```

Comment:
we dont need branch specifics for eco modules usually, they are not that often updated

Reply:
OK, but lets maybe not add more of those.